### PR TITLE
day1

### PR DESCRIPTION
**Pull request by ShayRasooly**

### CODE REVIEW

File: Week1/Day1/DailyChallenge/script.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# 1
+entered_string = input("Please enter a string of exactly ten characters: ")
```
Comment (ShayRasooly) [inline Week1/Day1/DailyChallenge/script.py:2]: maybe a bit better name for this variable, something that contains "input" perhaps

File: Week1/Day1/ExercisesXP/script.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
+# Exercise 1
+print("Hello world\n" * 4)
+
+# Exercise 2
+some_math = (99 ** 3) * 8
+print(some_math)
+
+# Exercise 3
+def predict(statement, prediction):
```
Comment (ShayRasooly) [inline Week1/Day1/ExercisesXP/script.py:9]: nice idea!!
maybe read a bit about "assert", function that does exactly this and widely used for tests

File: Week1/Day1/ExercisesXP/script.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    if statement == prediction:
+        print(f"{prediction} was the right prediction")
+    else:
+        print(f"{prediction} was the wrong prediction")
+
+predict(5 < 3, False)
+predict(3 == 3, True)
+predict(3 == "3", False)
+# "3" > 3 It is impossible to perform the greater than operator on a string and an integer
```
Comment (ShayRasooly) [inline Week1/Day1/ExercisesXP/script.py:18]: with assert you could've predicted errors as well ;)

File: Week1/Day1/ExercisesXP/script.py [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+predict("Hello" == "hello", False)
+
+# Exercise 4
+computer_brand = "Acer"
+if computer_brand[0] in 'AEIOYaeioy':
+    print(f'I have an {computer_brand} computer')
```
Comment (ShayRasooly) [inline Week1/Day1/ExercisesXP/script.py:24]: nice touch with the a and an
can you do it in one line?

File: Week1/Day1/ExercisesXPNinja/script.py [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+predict(y, False)
+
+print("x is", x)
+print("y is", y)
+print("a:", a)
+print("b:", b)
+
+# Exercise 4
+my_text = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
+print(len([character for character in my_text if character != ' ']))
```
Comment (ShayRasooly) [inline Week1/Day1/ExercisesXPNinja/script.py:28]: nice list comprehension!

File: Week1/Day1/DailyChallenge/script.py (hunk 1 above)
Comment (David-7805) [reply Week1/Day1/DailyChallenge/script.py:2]: You're right!

File: Week1/Day1/ExercisesXP/script.py (hunk 2 above)
Comment (David-7805) [reply Week1/Day1/ExercisesXP/script.py:9]: Interesting, when I have time I will read it.

File: Week1/Day1/ExercisesXP/script.py (hunk 3 above)
Comment (David-7805) [reply Week1/Day1/ExercisesXP/script.py:18]: Then that is a better function!

File: Week1/Day1/ExercisesXPNinja/script.py (hunk 5 above)
Comment (David-7805) [reply Week1/Day1/ExercisesXPNinja/script.py:28]: Thanks!

File: Week1/Day1/ExercisesXP/script.py (hunk 4 above)
Comment (David-7805) [reply Week1/Day1/ExercisesXP/script.py:24]: Yes, I now found out that this is possible with print functions as well : )

**Pull request merged**